source_code/main_mcu/src/TIMER/driver_timer.h

```diff
@@ -60,16 +60,27 @@ typedef enum {TIMER_EXPIRED = 0, TIMER_RUNNING = 1} timer_flag_te;
 
 #endif
 
+#define IS_LEAP_YEAR(year)  ((((year) % 4 == 0) && ((year) % 100 != 0)) || ((year) % 400 == 0))
```

Comment:
wow that's cool... where did you find this one?

Reply:
That is a very common construct that can be seen everywhere... :-)